docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -352,4 +352,46 @@ To avoid Neuron device out of memory, it's suggested to finish all base inferenc
 </Tip>
 
 
+## Latent Consistency Models
+
+Latent Consistency Models (LCMs) were proposed in [Latent Consistency Models: Synthesizing High-Resolution Images with Few-Step Inference by Simian Luo, Yiqin Tan, Longbo Huang, Jian Li, and Hang Zhao](https://huggingface.co/papers/2310.04378). LCMs enable inference with fewer steps on any pre-trained LDMs, including Stable Diffusion and SDXL.
+
+Here is an example to compile a LCM of Stable Diffusion XL: [latent-consistency/lcm-sdxl](https://huggingface.co/latent-consistency/lcm-sdxl) and run inference on AWS Inferentia 2 :
+
+### Compile LCM
+
+***Export via APIs***
+
+```python
+from optimum.neuron import NeuronStableDiffusionXLPipeline
+
+model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+unet_id = "latent-consistency/lcm-sdxl"
+num_images_per_prompt = 1
+input_shapes = {"batch_size": 1, "height": 1024, "width": 1024, "num_images_per_prompt": num_images_per_prompt}
+compiler_args = {"auto_cast": "matmul", "auto_cast_type": "bf16"}
```

Comment:
personal nit: If these are constants I would put them in upper case to highlight it 😊

Reply:
They are actually pretty much adjustable, I would keep them this way.